docs/workflow/building/libraries/README.md

```diff
@@ -200,7 +200,7 @@ Note that you cannot generally build native components for another OS but you ca
 ### Building in Release or Debug
 
 By default, building from the root or within a project will build the libraries in Debug mode.
-One can build in Debug or Release mode from the root by doing `./build.sh -subsetCategory libraries  -c Release` or `./build.sh -subsetCategory libraries -c Debug` or when building a project by specifying `/p:ConfigurationGroup=[Debug|Release]` after the `dotnet msbuild` command.
+One can build in Debug or Release mode from the root by doing `./build.sh -subsetCategory libraries  -c Release` or `./build.sh -subsetCategory libraries -c Debug` or when building a project by specifying `/p:Configuration=[Debug|Release]` after the `dotnet msbuild` command.
```

Comment:
```suggestion
One can build in Debug or Release mode from the root by doing `./build.sh -subsetCategory libraries  -c Release` or `./build.sh -subsetCategory libraries -c Debug`.
```

No need to specify again how to build an individual library with the dotnet CLI as we already showed that above and as we now use the default knobs used in any other customer project: `-c Debug/Release`.